components/Message.js

```diff
@@ -1,7 +1,8 @@
 import { Text } from "@asyncapi/generator-react-sdk";
-import { generateExample, getHeadersExamples, getPayloadExamples } from "@asyncapi/generator-filters";
```

Comment:
can you explain this change to a custom functions

Reply:
`getHeadersExamples` or `getPayloadExamples` only return the `examples[*].headers` or `examples[*].payload`, so we don't have corresponding `name` or `summary` for appropriate example. If we will have change from PR in `@asyncapi/generator-filters` we must know that then it's a breaking change and we must update check all templates where we use the `getHeadersExamples` and `getPayloadExamples`. Making changes only here is safe.

Reply:
`getHeadersExamples` or `getPayloadExamples` are more than what you wrote, if examples are not in `examples` object, they are taken from payload or headers schema (if provided of course). 

we can release generator-filters 2.0 🤷🏼 unless you already have your own versions of these functions in react component. Then we are just looking an effort we did in the past to move generic functions to `@asyncapi/generator-filters`

Reply:
> getHeadersExamples or getPayloadExamples are more than what you wrote, if examples are not in examples object, they are taken from payload or headers schema (if provided of course).

Functions in PR also are looking for that :)

> we can release generator-filters 2.0 🤷🏼 unless you already have your own versions of these functions in react component. Then we are just looking an effort we did in the past to move generic functions to @asyncapi/generator-filters

So I create PR in `generator-filters`.

Reply:
@derberg Created https://github.com/asyncapi/generator-filters/pull/41